### PR TITLE
Upgrade to ESlint version 3

### PR DESCRIPTION
**Pull request by vegardok**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/node_modules/
+node_modules/
```
Comment (thedotedge) [inline .gitignore:1]: If `node_modules` is now ignored, does it make sense to remove the folder as well?

Comment (thedotedge) [reply .gitignore:1]: Otherwise LGTM

**Comment by ivarconr** [comment]
Actually there is already a branch with eslint-3 also modularizing the project better : 
https://github.com/schibsted/eslint-config-spt/tree/v3.0.0

We have taken on to release this version soon!

**Comment by SimenB** [comment]
@vegardok See `npm dist-tag ls eslint-config-spt` or `npm dist-tag ls eslint-config-spt-react`

**Comment by SimenB** [comment]
Anyways, this change is already done in #12

**Pull request closed without merging**